ansible_collections/arista/avd/examples/single-dc-l3ls/playbook.yml

```diff
@@ -1,14 +1,14 @@
 ---
-- name: Run AVD
+- name: Run AVD # (1)!
```

Comment:
I understand why the comments are here but out of context it looks weird (though when included in the README.md it does make a lot of sense) - not sure if we could add some additional comment in this file explaining why the comments are here.

Reply:
I can add a comment at the root of the playbook file.